python/ray/serve/scripts.py

```diff
@@ -241,10 +281,15 @@ def run(config_or_import_path: str, address: str):
             cli_logger.print(
                 "Deploying function or class imported from " f"{config_or_import_path}."
             )
-            func_or_class = import_attr(config_or_import_path)
-            if not isinstance(func_or_class, Deployment):
-                func_or_class = serve.deployment(func_or_class)
-            func_or_class.deploy()
+
+            deployment = serve.deployment(name="run")(config_or_import_path)
+
+            serve.start()
```

Comment:
is this necessary here? it's not in here before

Reply:
Yes, I believe so. `serve run` uses the Python API directly. The other CLI commands ping the dashboard, which connects to Serve. Since `serve run` doesn't ping the dashboard, it needs to start Serve somehow.

Plus, without `detached=True`, `serve.start()` should tear down the deployment once `serve run` terminates.